examples/user-management/swift-user-management/.env.example

```
@@ -1,4 +1,4 @@
 # Get these from your API settings: https://supabase.com/dashboard/project/_/settings/api
 
 SUPABASE_URL=https://mysupabasereference.supabase.co
-SUPABASE_ANON_KEY=my.supabase.anon.key
\ No newline at end of file
+SUPABASE_PUBLISHABLE_KEY=my.supabase.anon.key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)